client/src/pages/Household.js

```diff
@@ -10,29 +12,125 @@ import ButtonContainer from '../components/ButtonContainer';
 import Form from '../components/Form';
 import Wrapper from '../components/Wrapper';
 import PageContainer from '../components/PageContainer';
+import styled from '@emotion/styled';
+
+const DeleteButton = styled.img`
+  display: flex;
+  justify-content: center;
+  width: 20px;
```

Comment:
it looks very good!))
just little thing that i've got in my reviews:
Elements like input or button should take the size of their context. That means you should rather give the element padding and font-size, but no width and no height. you can change their size later through the parent element, which makes it easier for you to change all elements if needed, or make it responsive.